style.css

```diff
@@ -0,0 +1,146 @@
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+:root {
+  --ff-inter: 'Inter', sans-serif;
+  --white: #fff;
+  --grey: #ebf0ee;
+  --veryLightGreen: #c5e9da;
+  --lime: #36b37f;
+  --green: #008552;
+  --darkBlue: #344563;
+  --darkerBlue: #172b4d;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  line-height: 1.5rem;
+  font-family: var(--ff-inter);
+  font-size: 1rem;
+  font-weight: 400;
+  color: var(--darkBlue);
+  background-color: var(--white);
+}
+
+h1,
+h2,
+h3,
+h4 {
+  color: var(--darkerBlue);
+  margin: 0;
+}
+
+a,
+a::before,
+a::after {
+  text-decoration: none;
+  color: inherit;
+}
+
+p {
+  margin-top: 0;
+}
+
+.container {
+  max-width: 90rem;
+}
```

Comment:
- [ ] Please consider removing this width on the `.container` element, this makes your design unresponsive on large devices such as laptops. Removing this CSS styling will make your webpage responsive across small and large screens.

Reply:
Will it be ok if I add a 'margin: auto;' so that the page can stay in the middle of the page on large devices?

Or should I remove the max width altogether?

Reply:
Hi @Timbar09 I think just remove the `max-width` while reviewing your project I noticed that on large screens it distorted the alignment of elements on the page. I believe your design will be much better without it.

![Screenshot from 2022-12-01 10-36-12](https://user-images.githubusercontent.com/16392046/204993661-5b45073a-0e3e-4dd0-9b53-1ac312535dba.png)

Reply:
Got it.